src/pkg/robot/model/robot.go

```diff
@@ -45,12 +45,13 @@ type RobotQuery struct {
 
 // RobotCreate ...
 type RobotCreate struct {
-	Name        string         `json:"name"`
-	ProjectID   int64          `json:"pid"`
-	Description string         `json:"description"`
-	Disabled    bool           `json:"disabled"`
-	Visible     bool           `json:"-"`
-	Access      []*rbac.Policy `json:"access"`
+	Name              string         `json:"name"`
+	ProjectID         int64          `json:"pid"`
+	Description       string         `json:"description"`
+	Disabled          bool           `json:"disabled"`
+	Visible           bool           `json:"-"`
+	ByPassPolicyCheck bool           `json:"-"`
```

Comment:
`ByPassPolicyCheck` maybe not needed here, the scanner can create robot account with (Resource, ScannerPull) access.

Reply:
removed